.github/workflows/c.yaml

```diff
@@ -59,13 +61,16 @@ jobs:
                 jpeg-turbo \
                 libtiff \
                 openjpeg \
+                jxrlib \
                 gdk-pixbuf \
                 libxml2 \
                 sqlite \
                 cairo \
                 glib \
                 xdelta \
                 doxygen
+            # jxrlib from homebrew has a bug
```

Comment:
Could you file a Homebrew issue or PR for this and add a link to it in a comment here?

Reply:
Please see https://github.com/Homebrew/homebrew-core/issues/117415 .